explorer/client/src/createStore.ts

```diff
@@ -6,7 +6,7 @@ import logger from 'redux-logger'
 import reducer from './reducers'
 
 let middleware: Middleware[] = [thunkMiddleware, createQueryMiddleware()]
-if (process.env.LOG_REDUX === 'true') {
+if ((process.env.LOG_REDUX || '').toLowerCase() === 'true') {
```

Comment:
Looks like this change got mixed up with the other PR of yours, should :scissors: this out